coalib/coala.py

```diff
@@ -42,18 +42,39 @@ def main():
             return mode_json(args)
 
         if args.show_bears:
-            from coalib.settings.ConfigurationGathering import (
-                get_filtered_bears)
-
-            local_bears, global_bears = get_filtered_bears(
-                args.filter_by_language, log_printer)
-
-            show_bears(local_bears,
-                       global_bears,
-                       args.show_description or args.show_details,
-                       args.show_details,
-                       console_printer)
-
+            if args.filter_by:
+                from coalib.parsing.Filter import Filter
+                filter = args.filter_by[0]
+                filter_args = args.filter_by[1:]
```

Comment:
> >  --filter-by language cpp --filter-by aspect License
>
> In case of multiple filter-by available, it only yields results for the later filter. I'll try to fix that.

That would be good to have definitely. But actually we can merge the more primitive version first and improve from that. Step by step is always good ;)

okay an "and" operator is anyway not really useful, but it's good that "or" is supported :+1: just an impression that we may want to support multiple operators.

> I've not thought of aliases, at the moment. But we just need to add an entry in the map for that. That's fairly easy to do.

:+1: